seqr/views/utils/permissions_utils.py

```diff
@@ -76,22 +76,26 @@ def _map_anvil_seqr_permission(anvil_permission):
 def anvil_has_perm(user, permission_level, project):
     if not project_has_anvil(project):
         return False
-    return has_workspace_perm(user, permission_level, project.workspace_namespace, project.workspace_name)
+    return bool(has_workspace_perm(user, permission_level, project.workspace_namespace, project.workspace_name))
 
 
-def has_workspace_perm(user, permission_level, namespace, name, can_share=False):
-    workspace_permission = user_get_workspace_access_level(user, namespace, name)
+def has_workspace_perm(user, permission_level, namespace, name, can_share=False, meta_fields=None):
+    workspace_permission = user_get_workspace_access_level(user, namespace, name, meta_fields) if meta_fields else \
+        user_get_workspace_access_level(user, namespace, name)
```

Comment:
I call `user_get_workspace_access_level()` with two different parameter lists because I don't want to change the test assertions about the function's parameters. There are many of these assertions which could be a distraction to review. After your reviewing, I could change it to `workspace_permission = user_get_workspace_access_level(user, namespace, name, meta_fields)` (without following `if`) and update the assertions.

Reply:
a cleaner way to do this that will not change how the existing calls are made is 
```
kwargs = {}
if meta_fields:
    kwargs['meta_fields'] = meta_fields
workspace_permission = user_get_workspace_access_level(user, namespace, name, **kwargs)
```